src/multi_vector_simulator/utils/constants_json_strings.py

```diff
@@ -83,6 +83,7 @@
 OPTIMIZE_CAP = "optimizeCap"
 INSTALLED_CAP = "installedCap"
 MAXIMUM_CAP = "maximumCap"
+MAXIMUM_ADD_CAP = "maximumAddCap"
```

Comment:
There is a section for parameters added in pre-processing further down, I think you should move the parameter there: https://github.com/rl-institut/multi-vector-simulator/blob/3e4840ee34157f14ea78a04e898e8bff4c4b2f3a/src/multi_vector_simulator/utils/constants_json_strings.py#L171